.github/workflows/publish-android.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Breez SDK to Breez Maven Repository
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'breez-sdk repo release (MAJOR.MINOR.PATCH)'
+        required: true
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install required dependencies
+        run: |
+          brew install protobuf
+          cargo install --version 0.22.0 uniffi_bindgen
```

Comment:
This will soon be gone. See https://github.com/breez/breez-sdk-swift/pull/3#discussion_r1167411885.